Podcast/InternalProfileViewController.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  InternalProfileViewController.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 3/7/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class InternalProfileViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, InternalProfileHeaderViewDelegate {
+    
+    var tableView: UITableView!
+    var internalProfileHeaderView: InternalProfileHeaderView!
+    
+    let sectionsAndItems = [
+        ["Listen History", "Downloads", "Subscriptions"],
+        ["Settings"]
+    ]
+    let reusableCellID = "profileLinkCell"
+    let sectionSpacing: CGFloat = 18
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else { return }
+        
+        navigationController?.setNavigationBarHidden(false, animated: false)
+        navigationItem.title = "Your Stuff"
+        
+        internalProfileHeaderView = InternalProfileHeaderView(frame: CGRect(x: 0, y: 0, width: view.frame.width, height: InternalProfileHeaderView.height))
+        internalProfileHeaderView.delegate = self
+        internalProfileHeaderView.setUser(System.currentUser)
+
+        tableView = UITableView(frame: CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height), style: .grouped)
+        tableView.backgroundColor = .podcastWhiteDark
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.showsVerticalScrollIndicator = false
+        tableView.contentInset = UIEdgeInsetsMake(0, 0, appDelegate.tabBarController.tabBarHeight, 0)
+        tableView.register(InternalProfileTableViewCell.self, forCellReuseIdentifier: reusableCellID)
+        tableView.tableHeaderView = internalProfileHeaderView
+        tableView.separatorInset = UIEdgeInsetsMake(0, 0, 0, 0)
+        view.addSubview(tableView)
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.setNavigationBarHidden(false, animated: true)
```

Comment:
you don't need to call this every time, I think its only necessary in viewdidload